scripts/cutover_to_pg.sh

```
@@ -0,0 +1,135 @@
+
+if [ "$HEALTH" -eq 200 ] && [ "$SEARCH" -eq 200 ] && [ "$LOGIN" -eq 422 ]; then
+  log "=== ALL SMOKE TESTS PASSED ==="
+  log "Cutover complete. App is running on PostgreSQL."
+  log ""
+  log "SQLite backup preserved at: $APP_DIR/.env.sqlite.backup"
+  log "SQLite file preserved at: /opt/whoknows/data/whoknows.db"
+  log ""
+  log "To rollback: bash scripts/rollback_to_sqlite.sh"
+else
+  log "=== SMOKE TESTS FAILED — ROLLING BACK ==="
+  ssh "$VM1_HOST" "cp $APP_DIR/.env.sqlite.backup $APP_DIR/.env && cd $APP_DIR && docker compose -f docker-compose.prod.yml up -d --remove-orphans"
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)